src/styles/main.scss

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  width: $calendar-size;
+  display: flex;
+  padding: $padding;
+  justify-content: left;
+  flex-wrap: wrap;
+  gap: $block-gap;
+  font-size: 30px;
+
+  @each $day, $offset in $days-of-week {
+    &--start-day-#{$day} &__day:first-child {
+      margin-left: calc(#{$offset} * ($block-size + $block-gap));
+    }
```

Comment:
The loop using '@each' is meant to set different 'margin-left' values for the first child of '.calendar__day' based on the day of the week. Make sure that the '$days-of-week' variable is correctly defined and contains the expected day and offset pairs.